### ISSUE TITLE
It's impossible to disable CyclomaticComplexity

### ISSUE THREAD

**Issue by swistak** [bbatsov/rubocop#5189]
It seems to be impossible to disable a CyclomaticComplexity filter.

## Expected behavior

It should be possible to disable a Cyclomatic Complexity Metric (in the same way PercivedComplexity is disabled.

## Actual behavior

It seems to be impossible to disable a Complexity metrics on the method.

## Steps to reproduce the problem

```
# CyclomaticComplexity Bug
class A
  # rubocop:disable Metrics/PerceivedComplexity Metrics/CyclomaticComplexity
  def complex_method(e = {}) # rubocop:disable Metrics/PerceivedComplexity Metrics/CyclomaticComplexity
    # rubocop:disable Metrics/PerceivedComplexity Metrics/CyclomaticComplexity
    if e['x']
      e['x']
    elsif e['y']
      if e['y']['a']
        e['y']['a']
      else
        e['y']['b']
      end
    elsif e['z']
      e['z']
    elsif e['x']
      e['x']
      if e['x']
        :z
      else
        :y
      end
    else
      e.first
    end
  end
end
```

## RuboCop version

```
$ rubocop -V
0.51.0 (using Parser 2.4.0.2, running on ruby 2.1.6 x86_64-linux)

```

**Comment by pocke** [comment]
We can disable an offense by the cop with a disable comment.

```ruby
# CyclomaticComplexity Bug
class A
  # rubocop:disable Metrics/PerceivedComplexity,Metrics/CyclomaticComplexity
  def complex_method(e = {})
    if e['x']
      e['x']
    elsif e['y']
      if e['y']['a']
        e['y']['a']
      else
        e['y']['b']
      end
    elsif e['z']
      e['z']
    elsif e['x']
      e['x']
      if e['x']
        :z
      else
        :y
      end
    else
      e.first
    end
  end
end
```


```bash
$ rubocop 
Inspecting 1 file
W

Offenses:

test.rb:3:1: W: Lint/MissingCopEnableDirective: Re-enable Metrics/PerceivedComplexity cop with # rubocop:enable after disabling it.
  # rubocop:disable Metrics/PerceivedComplexity,Metrics/CyclomaticComplexity
^
test.rb:4:3: C: Metrics/AbcSize: Assignment Branch Condition size for complex_method is too high. [16.16/15]
  def complex_method(e = {}) ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^
test.rb:4:3: C: Metrics/MethodLength: Method has too many lines. [20/10]
  def complex_method(e = {}) ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 3 offenses detected
```


In a disable comment, we can specify multiple cops. And the cops should be separated with a comma, not a space.
So the disable comment does not work in your example.

**Comment by swistak** [comment]
/me faceslap.

Thank you for the help and sorry to have wasted your time.

**Comment by bbatsov** [comment]
Seems we should add some cop for that. 😉
